Backendless/Transaction/TransactionHelper.cs

```diff
@@ -0,0 +1,199 @@
+﻿using BackendlessAPI.Exception;
+using System;
+using System.Collections.Generic;
+using System.Reflection;
+
+namespace BackendlessAPI.Transaction
+{
+  class TransactionHelper
+  {
+    private static String LAST_LOGIN_COLUMN_NAME = "lastLogin";
+    private static String PASSWORD_KEY = "password";
+    private static String SOCIAL_ACCOUNT_COLUMN_NAME = "socialAccount";
+    private static String USER_STATUS_COLUMN_NAME = "userStatus";
+
+    internal static void RemoveSystemField( Dictionary<String, Object> changes )
+    {
+      changes.Remove( LAST_LOGIN_COLUMN_NAME );
+      changes.Remove( PASSWORD_KEY );
+      changes.Remove( SOCIAL_ACCOUNT_COLUMN_NAME );
+      changes.Remove( USER_STATUS_COLUMN_NAME );
+      changes.Remove( "objectId" );
+      changes.Remove( "created" );
+      changes.Remove( "updated" );
+    }
+
+    static internal OpResult MakeOpResult( String tableName, String operationResultId, OperationType operationType )
+    {
+      return new OpResult( tableName, operationResultId, operationType );
+    }
+
+    internal static List<Object> ConvertMapsToObjectIds( List<Dictionary<String, Object>> objectMaps )
+    {
+      List<Object> objectIds = new List<Object>();
+
+      foreach( Dictionary<String, Object> objectMap in objectMaps )
+        objectIds.Add( ConvertObjectMapToObjectIdOrLeaveReference( objectMap ) );
+
+      return objectIds;
+    }
+    
+    internal static Dictionary<String, Object> ConvertInstanceToMap<E>( E instance )
```

Comment:
where is this used?

Reply:
@markpiller this method is used to create a result when working with instances